File: java/3_operator/logika/And.java

class And {
    
    public static void main(String[] args){
        boolean a = true;
        boolean b = false;

        boolean logikaAnd_11 = a && a;
        boolean logikaAnd_10 = a && b;
        boolean logikaAnd_01 = b && a;
        boolean logikaAnd_00 = b && b;

        System.out.println("Hasil logika AND true-true adalah: " + logikaAnd_11);
        System.out.println("Hasil logika AND true-false adalah: " + logikaAnd_10);
        System.out.println("Hasil logika AND false-true adalah: " + logikaAnd_01);
        System.out.println("Hasil logika AND false-false adalah: " + logikaAnd_00);
    }
}

File: kotlin/3_operator/logika/And.kt

fun main() {
    
    val a: Boolean = true
    val b: Boolean = false

    val logikaAnd_11 = a && a
    val logikaAnd_10 = a && b
    val logikaAnd_01 = b && a
    val logikaAnd_00 = b && b

    println("Hasil logika AND true-true adalah:     $logikaAnd_11")
    println("Hasil logika AND true-false adalah:    $logikaAnd_10")
    println("Hasil logika AND flase-true adalah:    $logikaAnd_01")
    println("Hasil logika AND false-false adalah:   $logikaAnd_00")
}
